pkg/adapter/mtping/runner_test.go

```diff
@@ -44,7 +47,37 @@ func TestAddRunRemoveSchedule(t *testing.T) {
 
 	entry.Job.Run()
 
-	validateSent(t, ce, `{"body":"some data"}`)
+	validateSent(t, ce, `{"body":"some data"}`, nil)
+
+	runner.RemoveSchedule(entryId)
+
+	entry = runner.cron.Entry(entryId)
+	if entry.ID == entryId {
+		t.Error("Entry has not been removed")
+	}
+}
+func TestAddRunRemoveScheduleWithExtensionOverride(t *testing.T) {
+	logger := logtesting.TestLogger(t)
+	ce := adaptertesting.NewTestClient()
+
+	runner := NewCronJobsRunner(ce, logger)
+
+	wantExtensions := map[string]string{"1": "one", "2": "two"}
+	entryId, err := runner.AddSchedule("test-ns", "test-name", "* * * * ?", "some data",
+		"a sink", &v1.CloudEventOverrides{Extensions: wantExtensions})
+
+	if err != nil {
+		t.Errorf("Should not throw error %v", err)
+	}
+
+	entry := runner.cron.Entry(entryId)
+	if entry.ID != entryId {
+		t.Error("Entry has not been added")
+	}
+
+	entry.Job.Run()
+
+	validateSent(t, ce, `{"body":"some data"}`, wantExtensions)
```

Comment:
This looks identical to the test above, how about turning it into a table test instead? One with extensions, one without.